### ISSUE TITLE
Old url doesn't work with the new documentation

### ISSUE THREAD

**Issue by ice-blaze** [oblique-bit/oblique#131]
### Oblique version

11

### Description

```true
The old URL of the documentation https://oblique.bit.admin.ch/oblique doesn't work with the new documentation which create a white page. Could be disturbing but can be fixed with a refresh.
```

### Location

https://oblique.bit.admin.ch/oblique

### Current content

```true
Only discus is visible but the content is empty.
```

### Proposed solution

```true
Redirection previous URL to this page https://oblique.bit.admin.ch/introductions/welcome
```

**Comment by gillerr** [comment]
Will be done with Oblique 12

**Comment by gillerr** [comment]
this has been fixed with 12.0.0
